bsconfig.json

```diff
@@ -1,14 +1,19 @@
 {
   "name": "@highpoint/bs-js-fetch",
+  "bs-dev-dependencies": ["bs-tape"],
   "sources": [
     {
       "dir": "src"
+    }, 
+    {
+      "dir": "tests",
+      "type": "dev"
     }
   ],
   "refmt": 3,
   "suffix": ".bs.js",
   "package-specs": {
-    "module": "es6",
+    "module": "commonjs",
```

Comment:
Why switch from ES6 to CommonJS?

Reply:
Built versions only matter for tests, and I just wanted to have it running as quickly as possible. I could've used no bundler, but the package I'm using to run tests in the browser adds the stdin as a normal script tag (https://github.com/juliangruber/browser-run/blob/master/static/index.html) so I needed to use a bundler and went with browserify which is the simplest to setup. If I wanted to keep es6 I would've had to use a browserify transform, felt simpler/faster to just switch to commonjs instead.